docs/topics/proc_showing-debug-logging-on-openshift.adoc

```diff
@@ -0,0 +1,116 @@
+
+[#showing-debug-logging-on-openshift]
+= Showing Debug Logging on OpenShift
+
+Start your application and interact with it to see the debugging statements in OpenShift.
+
+.Prerequisites
+
+* A Maven-based application
+ifndef::wf-swarm[with debug logging enabled]
```

Comment:
This conditional sounds wrong.

Reply:
It's correct AFAICT; in WFS, I operate with the assumption that debug logging is not enabled, and use the `-Dswarm.logging=DEBUG` ad hoc setting, which IMO makes more sense for debugging as it is usually a temporary thing.

Reply:
OK, so, well, I don't see the corresponding `endif` :-) Also I'd guess that WildFly Swarm also requires `oc` installed and authenticated.

Reply:
If you include the conditional text in the square brackets, it's a one-liner without the `endif`.

Reply:
Aaaaa :-) Sorry for the noise.